src/sentry/utils/snuba.py

```diff
@@ -835,7 +836,7 @@ def bulk_snuba_queries(
             request.tenant_ids = request.tenant_ids or dict()
             request.tenant_ids["referrer"] = referrer
 
-    params: SnubaQuery = [(request, lambda x: x, lambda x: x) for request in requests]
+    params = [(request, lambda x: x, lambda x: x) for request in requests]
```

Comment:
what are the lambdas for?

Reply:
not sure because they were always there but they are "translators" a part of a query body often referred to throughout the codebase as  "forward" and "reverse"